build.zig

```diff
@@ -88,6 +100,9 @@ pub fn build(b: *Builder) !void {
         inline for (&[_][]const u8{ "user_program_data", "user_program" }) |user_program| {
             // Add some test files for the user mode runtime tests
             const user_program_step = b.addExecutable(user_program ++ ".elf", null);
+            user_program_step.addPackage(kernel_pkg);
+            user_program_step.addPackage(arch_mock);
+            user_program_step.addPackage(arch_pkg);
```

Comment:
I don't think the user programs will need the packages as they're just assembly files.

Reply:
Yeah I wasn't sure what the code was doing so I put them in just in case.